loops.js

```diff
@@ -5,20 +5,28 @@
 // Hoera!
 // Hoera!
 // ==========================================
-
-
+for (i=0; i<3; i++) {
+    console.log("Hoera")
+}
+console.log("\n"); //print lege ruimte tussen uitkomsten in de console
 
 // ==========================================
 // Opdracht 2. Maak een for-loop die 4 keer het woord "loop..." logt, en bij de vijfde loop het woord "klaar!"
 // Verwachte uitkomsten:
-// loop...
+// loop.
 // loop...
 // loop...
 // loop...
 // klaar!
  // ==========================================
-
-
+for (i=0; i<5; i++) {
+    if (i==4) {
```

Comment:
bij een vergelijking is de 3 x 'is' (===), te verkiezen boven de 2x omdat de vergelijking dan ook vergelijkt op type variabele, in dit geval bevat het beide een nummer (zie het tabel in 2.7 van edhub).